plugin/trino-bigquery/src/test/java/io/trino/plugin/bigquery/BigQueryQueryRunner.java

```diff
@@ -118,7 +118,7 @@ public void createDataset(String datasetName)
             bigQuery.create(dataset);
         }
 
-        public void dropDataset(String dataset)
+        public void dropDatasetIfExist(String dataset)
```

Comment:
Thanks for making the semantics clear from the name itself.

nit: `dropDatasetIfExists` to match the SQL `DROP DATASET IF EXISTS`.